desktop/cmp/form/inputs/Select.scss

```diff
@@ -86,14 +86,15 @@
     &__option {
       &--is-selected {
         color: var(--xh-text-color);
-        background: var(--xh-bg-alt);
+        background: var(--xh-select-selected);
```

Comment:
Do we *need* to declare new vars for this? Wouldn't switching the existing selected / focused styles be sufficient?
I'm wary of adding further vars if not necessary.